drf_fsm_transitions/viewset_mixins.py

```diff
@@ -6,7 +6,7 @@ def get_transition_viewset_method(transition_name, **kwargs):
     '''
     Create a viewset method for the provided `transition_name`
     '''
-    @action(methods=['post'], detail=True, **kwargs)
+    @action(methods=['post'], detail=True, url_name=transition_name, url_path=transition_name, **kwargs)
```

Comment:
This change is also unneeded.

Reply:
Its needed so that the reverse names appear correctly